src/python/director/cameracontrolpanel.py

```diff
@@ -5,6 +5,7 @@
 from director import cameracontrol
 from director import propertyset
 from director import pointpicker
+from director import roboturdf
```

Comment:
the video mentioned above reminds me that I split off the maps server from OH so it can be used in Director outside of the OH stack. 

The Cmake still refers to it as "DRC_MAPS" I think